agent.py

```diff
@@ -152,6 +160,7 @@ def update_target_net(self):
     # Save model parameters on current device (don't move model between devices)
     def save(self, path, name='model.pth'):
         torch.save(self.online_net.state_dict(), os.path.join(path, name))
+        torch.save((self.online_net.phi, self.online_net.feature_size), os.path.join(path, 'phi_'+name))
```

Comment:
It was easy to forget to create the tuple here. Maybe a wrapper would save us in the future.

Reply:
Yes, agreed.